src/renderer/MenuRenderer.cpp

```
@@ -1,4 +1,5 @@
 #include "MenuRenderer.h"
+#include "MenuItem.h"
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)